<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IterativeReconstructionAppendParamsControlWidget</class>
 <widget class="QWidget" name="IterativeReconstructionAppendParamsControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IterativeReconstructionAppendParams</string>
  </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>maxPostAppend</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="maxPostAppendSpinBox">
       <property name="toolTip">
         <string>Maximal post-append iterations</string>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>inlierP3PThreshold</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="inlierP3PThresholdSpinBox">
       <property name="toolTip">
         <string>Inlier threshold for 3P->pose</string>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>maxP3PIterations</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="maxP3PIterationsSpinBox">
       <property name="toolTip">
         <string>Maximal ransac iterations for 3P->pose</string>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>inlierP6PThreshold</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="inlierP6PThresholdSpinBox">
       <property name="toolTip">
         <string>Inlier threshold for 6P->pose/3P->orientation</string>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>maxP6PIterations</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="maxP6PIterationsSpinBox">
       <property name="toolTip">
         <string>Maximal ransac iterations for 6P->pose/3P->orientation</string>
       </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
       <property name="text">
         <string>gammaP6P</string>
       </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="gammaP6PSpinBox">
       <property name="toolTip">
         <string>Target error probability for 6P->pose/3P->orientation</string>
       </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
       <property name="text">
         <string>speculativity</string>
       </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="speculativitySpinBox">
       <property name="toolTip">
         <string>This defines how many multicameras are subject for P3P evaluation at each iteration</string>
       </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
       <property name="text">
         <string>minimalInlierCount</string>
       </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="minimalInlierCountSpinBox">
       <property name="toolTip">
         <string>Minimal inlier count for hypotheis acceptance</string>
       </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
       <property name="text">
         <string>maximalFailureProbability</string>
       </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="maximalFailureProbabilitySpinBox">
       <property name="toolTip">
         <string>Maximal failure probability for hypothesis acceptance</string>
       </property>
     </widget>
    </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>maxPostAppendSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IterativeReconstructionAppendParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>inlierP3PThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IterativeReconstructionAppendParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>maxP3PIterationsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IterativeReconstructionAppendParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>inlierP6PThresholdSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IterativeReconstructionAppendParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>maxP6PIterationsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IterativeReconstructionAppendParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>gammaP6PSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IterativeReconstructionAppendParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>speculativitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IterativeReconstructionAppendParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>minimalInlierCountSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IterativeReconstructionAppendParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>maximalFailureProbabilitySpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IterativeReconstructionAppendParamsControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
